apps/pm-user/src/components/atoms/index.js

```diff
@@ -2,6 +2,7 @@ import Logo from './logo/';
 import Input from './input';
 import Button from './button';
 export { default as Menu } from './menu';
+export { default as Card } from './card';
 export * from "./icon";
 
 export { Button, Input, Logo };
```

Comment:
Choosing one way to export makes the code look good in my opinion. Otherwise, it is okay.

Reply:
that other way has to be removed